src/index.js

```diff
@@ -71,6 +72,10 @@ export default function (debug = printDebug) {
       })
     }
 
+    function buildAssertionsOutOfWrapperObject(wrapperObject) {
+      Object.keys(wrapperObject).forEach(key => overwriteMethod(wrapperObject[key], key))
```

Comment:
It's not supported in pre-ES5 engines, and it's also broken in a few engines, and most of the ES5 ones don't fully comply with ES6.

It's not a huge deal since this is a test framework but imo the fewer implicit environment dependencies it has, the better.